tests/Kernel/Paragraphs/MediaParagraphsTest.php

```diff
@@ -217,11 +209,16 @@ public function testTextWithMedia(): void {
     $paragraph->set('field_oe_media', ['target_id' => $media->id()]);
     $paragraph->save();
 
-    $html = $this->renderParagraph($paragraph);
-    $crawler = new Crawler($html);
     // Assert remote video is rendered properly.
-    $video_iframe = $crawler->filter('div.ecl-media-container__media--ratio-16-9 iframe');
-    $this->assertContains('I-163162', $video_iframe->attr('src'));
+    $html = $this->renderParagraph($paragraph);
+    $expected_values = [
+      'title' => NULL,
+      'caption' => 'Caption',
+      'text' => NULL,
+      'video' => '<iframe id="videoplayerI-163162" src="//ec.europa.eu/avservices/play.cfm?ref=I-163162&amp;lg=EN&amp;sublg=none&amp;autoplay=true&amp;tin=10&amp;tout=59" frameborder="0" allowtransparency allowfullscreen webkitallowfullscreen mozallowfullscreen width="640" height="390" class="media-avportal-content"></iframe>',
```

Comment:
There is a difference how media_avportal renders iframe.
Version 8.0-beta9 and higher:
```
'<iframe id="videoplayerI-163162" src="//ec.europa.eu/avservices/play.cfm?ref=I-163162&amp;lg=EN&amp;sublg=none&amp;autoplay=true&amp;tin=10&amp;tout=59" frameborder="0" allowtransparency allowfullscreen webkitallowfullscreen mozallowfullscreen width="640" height="390" class="media-avportal-content"></iframe>'
```
Version 8.0-beta6:
```
'<iframe id="videoplayerI-163162" src="//ec.europa.eu/avservices/play.cfm?ref=I-163162&amp;lg=EN&amp;sublg=none&amp;autoplay=true&amp;tin=10&amp;tout=59" frameborder="0" allowtransparency allowfullscreen webkitallowfullscreen mozallowfullscreen width="640" height="390"></iframe>'
```